test/testdrive/structured-errors.td

```diff
@@ -18,8 +18,16 @@
     PASSWORD SECRET pgpass
   )
 
-# TODO(guswynn): make testdrive support `startswith`, and hints and details
+# TODO(guswynn): make testdrive support `startswith`
 ! CREATE SOURCE "test_error"
   FROM POSTGRES CONNECTION pg (PUBLICATION 'mz_source')
   FOR ALL TABLES
 exact:failed to fetch publication information from PostgreSQL database
+detail:error connecting to server: Connection refused
```

Comment:
As Buildkite reports the "Connection refused" part is OS-specific. Consider matching only on `error connecting to server`

Reply:
Will do!